connector/src/main/java/org.apache.flink/connector/nebula/table/NebulaDynamicTableFactory.java

```diff
@@ -63,6 +63,12 @@ public class NebulaDynamicTableFactory implements DynamicTableSourceFactory,
             .noDefaultValue()
             .withDescription("the nebula graph space name.");
 
+    public static final ConfigOption<String> TABLE_NAME = ConfigOptions
+            .key("table-name")
+            .stringType()
+            .noDefaultValue()
+            .withDescription("the nebula graph tag or edge name.");
+
     public static final ConfigOption<String> LABEL_NAME = ConfigOptions
```

Comment:
what's the difference between TABLE_NAME and LABEL_NAME

Reply:
Thanks for the reminder, both fields actually function the same, didn't notice it before